compiler/src/main/java/io/neow3j/compiler/Compiler.java

```diff
@@ -407,7 +407,12 @@ private AbstractInsnNode handleInsn(NeoMethod neoMethod, AbstractInsnNode insn)
             // region ### ARRAYS ###
             case NEWARRAY:
             case ANEWARRAY:
-                neoMethod.addInstruction(new NeoInstruction(OpCode.NEWARRAY));
+                if (((IntInsnNode)insn).operand == 8) {
```

Comment:
I think it's fine like this, for the moment. 👍  💯

Reply:
If you feel like enhancing a bit the code, you could create more descriptive methods, like this: .isByteArray(insn)